docs/Tutorial.md

```diff
@@ -156,16 +156,9 @@ plug-in, is to right-click and select Document Properties.
 [Jules Verne text](https://github.com/py-pdf/fpdf2/raw/master/tutorial/20k_c1.txt)
 
 The key difference from the previous tutorial is the use of the 
-[accept_page_break](fpdf/fpdf.html#fpdf.fpdf.FPDF.accept_page_break) and the set_col methods.
+[`text_columns`](fpdf/fpdf.html#fpdf.fpdf.FPDF.text_column) method. 
```

Comment:
In PR #703, where we introduced tables, we removed the old explanations from all `docs/Tutorial*.md` files,
and replaced them by a text indicating that this section needed a new translation:

> This section has changed a lot and requires a new translation: https://github.com/PyFPDF/fpdf2/issues/267

Maybe we should do the same there?

I'd be happy to help and do so in an extra PR after this one is merged, if that's OK with you @gmischler

Reply:
I was planning to post an issue asking for updated translations. Removing the section entirely is more radical than that... 
Feel free to go ahead with it.